instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/attributes.go

```diff
@@ -56,11 +56,19 @@ func RequestIDAttr(requestID string) attribute.KeyValue {
 
 // DefaultAttributeSetter checks to see if there are service specific attributes available to set for the AWS service.
 // If there are service specific attributes available then they will be included.
+//
+// Deprecated: Use DefaultAttributeBuilder instead. This will be removed in a future release.
 func DefaultAttributeSetter(ctx context.Context, in middleware.InitializeInput) []attribute.KeyValue {
+	return DefaultAttributeBuilder(ctx, in, middleware.InitializeOutput{})
+}
+
+// DefaultAttributeBuilder checks to see if there are service specific attributes available to set for the AWS service.
+// If there are service specific attributes available then they will be included.
+func DefaultAttributeBuilder(ctx context.Context, in middleware.InitializeInput, out middleware.InitializeOutput) []attribute.KeyValue {
```

Comment:
This is not mentioned in changelog.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6543/commits/231e92b01921f58ff0d10ba02324684c7e2e5134